CHANGES.rst

```diff
@@ -59,6 +59,8 @@ Bugfixes
 - Fix editing service API calls using the service token (:pr:`5170`)
 - Fix excessive retries for Celery tasks with a retry wait time longer
   than 1 hour (:pr:`5172`)
+- Fix published editable files only being visible to users with access to the editing
+  timeline (:pr:`5218`)
```

Comment:
Add this for 3.1.1; looks like I just forgot to add the changelog stub for that after releasing 3.1 - maybe you could include a commit for this in your PR? See ef52ea02e6bf2 for the structure